src/App.vue

```diff
@@ -66,7 +66,13 @@ export default {
       this.todos.push(todo)
       this.comment = ''
     },
-  },
+    deleteWork(todo_id){
+      this.todos.splice(todo_id,1);
```

Comment:
`,`の後ろにはスペースをあけましょう。
他の箇所はできていますので、統一すると良いです。
```suggestion
      this.todos.splice(todo_id, 1);
```